pom.xml

```diff
@@ -112,6 +112,7 @@
     <module>ldap-api</module>
     <module>ldap-authenticator</module>
     <module>ldap-ui</module>
+    <module>application-ldap-cleanup</module>
```

Comment:
Would be better to be consistent with other modules names IMO so I would go for `ldap-user-cleanup`. In any case you should really name the folder like the artifact id (which is not the case right now).

Reply:
ok, done